src/components/Header.jsx

```diff
@@ -0,0 +1,48 @@
+import { Link, NavLink, useLocation } from 'react-router-dom';
+import styles from './Header.module.css';
+import {ReactComponent as UserIcon} from 'assets/imgs/user_icon.svg';
+import { useState } from 'react';
+
+const Header = () => {
+  const [isLogin, setIsLogin] = useState(true);
+  const location = useLocation();
+
+  const isFreeBoardPath = location.pathname === '/free';
+  const isUsedMarketPath = location.pathname === '/items' || location.pathname === '/additem';
+
+  return (
+      <header className={styles['header']}>
+        {isLogin ?
+          <div className={styles['header-container']}>
+            <Link to="/"><img className={styles['header-logo']} src='assets/imgs/logo_sm.svg' alt='로고'/></Link>
+            <nav className={styles['header-nav']}>
+              <NavLink 
+                to="/free"
+                className={
+                  isFreeBoardPath ? `${styles['nav-link']} ${styles['active']}` : styles['nav-link']
+                } 
+              >
```

Comment:
P1:
NavLink 태그에서 to 값과 현재 페이지가 일치하는지에 대한 값을 제공합니다.
아래처럼 사용하시는 것을 추천드립니다.

```suggestion
              <NavLink 
                to="/free"
                className={({ isActive })
                  isActive ? `${styles['nav-link']} ${styles['active']}` : styles['nav-link']
                } 
              >
```

*react-router-dom NavLink
https://reactrouter.com/en/main/components/nav-link